### PR TITLE
meta-iotqa: Ignore net interface name changing error

### PR DESCRIPTION
**Pull request by skuusela**
This error is random and doesn't seem to affect anything.

Signed-off-by: Simo Kuusela <simo.kuusela@intel.com>

### CODE REVIEW

**Comment by mythi** [comment]
test this please

File: meta-iotqa/lib/oeqa/runtime/sanity/baseos.py [hunk 1]
```diff
@@ -112,7 +112,8 @@ def test_baseos_systemd_boot_error(self):
             "Direct firmware load for i915/bxt_dmc_ver1_07.bin failed",
             "Direct firmware load for iwlwifi-8000C",
             "ACPI Error: Could not enable RealTimeClock event",
-            "hci_intel: probe of INT33E1:00 failed with error -2"
+            "hci_intel: probe of INT33E1:00 failed with error -2",
+            "Error changing net interface name 'usb0' to"
```
Comment (mythi) [inline meta-iotqa/lib/oeqa/runtime/sanity/baseos.py]: Apparently this is a race condition where someone takes ```usb0``` before systemd-udev gets to change it. What could that be?

**Comment by okartau** [comment]
test this please, likely CI related error

**Comment by skuusela** [comment]
test this please

**Comment by okartau** [comment]
test this please

**Pull request merged**